lib/ohai/plugins/linux/platform.rb

```diff
@@ -113,6 +129,15 @@ def os_release_file_is_cisco?
       # no way to determine platform_version in a rolling release distribution
       # kernel release will be used - ex. 3.13
       platform_version `uname -r`.strip
+    elsif (os_release_info = os_release_file_is_cisco?)
+      # Cisco platform not based on known distro above
+      if os_release_info['ID'] == 'nexus'
+        platform 'nexus'
+        platform_family 'cisco-wrlinux'
+      elsif os_release_info['ID_LIKE'] =~ /cisco-wrlinux/
+        platform_family 'cisco-wrlinux'
```

Comment:
Is this a breaking change? It looks like we used to get wrlinux before, now we're explicitly setting Cisco-wrlinux.

Is this platform family so unique that it needs to be its own family? Can it stay wrlinux or whatever wrlinux is based on?